homeassistant/components/niko_home_control/light.py

```diff
@@ -62,6 +68,9 @@ def __init__(self, light, data):
         self._unique_id = f"light-{light.id}"
         self._name = light.name
         self._state = light.is_on
+        self._brightness = None
+        if light._state["type"] == 2:
+            self._attr_supported_color_modes.add(ColorMode.BRIGHTNESS)
```

Comment:
Since `_attr_supported_color_modes` is no longer static, you should remove it from the class attributes on line 62 and set it only here.
```suggestion
        self._attr_supported_color_modes = {ColorMode.ONOFF}
        if light._state["type"] == 2:
            self._attr_supported_color_modes.add(ColorMode.BRIGHTNESS)
```